src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -34,14 +34,12 @@ export const getters = {
                         quarter: undefined
                     } : null;
             case (DataType.Output):
-                return rootState.modelRun.result ?
-                    {
-                        ...rootState.modelRun.result.filters,
```

Comment:
There's been a regression in hintr and these filters are not actually present (on master as well as this branch). We should re-add once they are back in hintr, thought they will likely be under a different property name (something like `modelRun.result.choropleth.filters`)

Reply:
No, separate.